src/components/Modals/TreasurySpendingMotionModal.js

```diff
@@ -0,0 +1,103 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import PropTypes from 'prop-types';
+
+import { useDispatch } from 'react-redux';
+import ModalRoot from './ModalRoot';
+import Button from '../Button/Button';
+import { TextInput } from '../InputComponents';
+import { congressActions } from '../../redux/actions';
+import { valueToBN } from '../../utils/walletHelpers';
+import { isValidSubstrateAddress } from '../../utils/bridge';
+
+import styles from './styles.module.scss';
+
+function TreasurySpendingMotionModal({ closeModal }) {
+  const dispatch = useDispatch();
+
+  const {
+    handleSubmit,
+    register,
+    formState: { errors },
+  } = useForm({
+    mode: 'all',
+    defaultValues: {
+      spending: 'congressSendLlm',
+    },
+  });
+
+  const onSubmit = ({ transferToAddress, transferAmount }) => {
+    dispatch(
+      congressActions.congressSendTreasuryLld.call({
+        transferToAddress,
+        transferAmount,
```

Comment:
This is wrong.

`transferAmount` is in LLD - so it's a string like `100.123`.
Sagas pass this directly to nodeRpcCall and nodeRpcCall passes it directly to polkadot-js-api, which directly passes it to blockchain.
So this will transfer `0.0000000001` LLD instead of the requested `100.123`.
You need `dollarsToGrains()` call here.

Reply:
Also I see that we've missed the same error in previous reviews. Can you fix this in `SpendingMotionModal.js` as well? Can be in this PR.